packages/core/types/index.js

```diff
@@ -57,12 +57,25 @@ export type EnvironmentContext =
   | 'service-worker'
   | 'node'
   | 'electron';
-export type Environment = {
+
+export type EnvironmentOpts = {
```

Comment:
Minor nitpick, but I noticed a discrepancy between Environment**Opts** and Dependency**Options**. Should we pick one or the other?